twa

```diff
@@ -440,6 +440,16 @@ function stage_5_robots_check {
   code=$(fetch_respcode "${domain}/robots.txt")
   if [[ "${code}" -eq 200 ]]; then
     PASS "Site provides robots.txt at: ${domain}/robots.txt"
+    robots=$(fetch -L -s "${domain}/robots.txt")
+
+    # set nocasematch option
+    shopt -s nocasematch
```

Comment:
Go ahead and put this at the top of the file (above `TWA_VERSION`) -- it'll make some of our other regex checks at little more resilient (also, no need for the comment) 🙂

Reply:
Moved to the top